src/ex4/apiTest.http

```diff
@@ -0,0 +1,30 @@
+# @name healthCheck
```

Comment:
not sure what this file for?

Reply:
Good morning this file is part of vsc addon named 'rest client'. This addon lets you send and receive http requests inside vsc, (even save jwt auth in header bearer)